Cargo.toml

```diff
@@ -15,7 +15,7 @@ license = "MIT OR Apache-2.0"
 bdk-macros = "^0.4"
 log = "^0.4"
 miniscript = "5.1"
-bitcoin = { version = "^0.26", features = ["use-serde"] }
+bitcoin = { version = "^0.26", features = ["use-serde", "base64"] }
```

Comment:
If you want to use a feature that is only available from `0.26.2` I think you'll have to change the version accordingly. I think `~0.26.2` is what we want here: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#tilde-requirements